scour/scour.py

```diff
@@ -2880,6 +2880,14 @@ def scourString(in_string, options=None):
    global numBytesSavedInTransforms
    doc = xml.dom.minidom.parseString(in_string)
 
+   # determine number of flowRoot elements in input document
+   # flowRoot elements don't render at all on current browsers (04/2016)
+   cnt_flowText_el = len(doc.getElementsByTagName('flowRoot'))
+   if cnt_flowText_el:
+      print("SVG input document uses {} flow text elements, which won't render on browsers!".format(cnt_flowText_el))
+      if options.error_on_flowtext:
+         sys.exit(1)
+
```

Comment:
In general, I agree.

In this case, why do all the hard work processing/cleansing the SVG, when we bail out anyway.